client/my-sites/themes/helpers.js

```diff
@@ -31,22 +26,30 @@ function appendActionTracking( option, name ) {
 	} );
 }
 
-export function getAnalyticsData( path, tier, site_id ) {
-	let basePath = sectionify( path );
+export function getAnalyticsData( path, { filter, vertical, tier, site_id } ) {
+	let analyticsPath = '/themes';
 	let analyticsPageTitle = 'Themes';
 
+	if ( vertical ) {
+		analyticsPath += '/:vertical';
+	}
+
 	if ( tier ) {
-		basePath += '/type/:tier';
+		analyticsPath += '/:tier';
+	}
+
+	if ( filter ) {
+		analyticsPath += '/filter/:filter';
 	}
 
 	if ( site_id ) {
-		basePath += '/:site_id';
+		analyticsPath += '/:site_id';
```

Comment:
I think we're trying to normalize `:site` for the site fragment

Reply:
Updated on 81615bfb0b21c6f4191241dc4c3f37abd3494d8f.